app/src/main/java/com/courseproject/tindar/entities/Conversation.java

```diff
@@ -0,0 +1,56 @@
+package com.courseproject.tindar.entities;
+
+import java.util.ArrayList;
+
+public abstract class Conversation {
+    private String users1;
+    private String users2;
```

Comment:
since there is single user1 and single user2, and we use user1 and user2 instead of users1 and users2?